src/org.xtuml.bp.core/src/org/xtuml/bp/core/common/PersistenceManager.java

```diff
@@ -856,22 +858,37 @@ static public List findAllComponents(ModelRoot modelRoot, Class elementClass) {
           return results;
         }
     }
-    
+
     static public void ensureAllInstancesLoaded(ModelRoot modelRoot,
             Class elementClass) {
-            List comps = findAllComponents(modelRoot, elementClass);
-            for (Iterator iter = comps.iterator(); iter.hasNext();) {
-                PersistableModelComponent component = (PersistableModelComponent) iter
-                        .next();
-                if (!component.isLoaded()) {
-                    try {
-                        component.load(new NullProgressMonitor());
-                    } catch (CoreException e) {
-                        CorePlugin.logError("Can't load component", e);
-                    }
-                }
-            }
-        }
+    	ensureAllInstancesLoaded(modelRoot, elementClass, null);
+    }
+    
+	static public void ensureAllInstancesLoaded(ModelRoot modelRoot, Class elementClass,
```

Comment:
The change is to prevent loading external systems if inter project referencing is not enabled.  I did have the preference wrong and changed it to: BridgePointProjectReferencesPreferences.BP_PROJECT_REFERENCES_ID